### PR TITLE
[WIP] Upgrade to Ember 1.13.9

### PR DESCRIPTION
**Pull request by elwayman02**
This commit upgrades ember-cli-github-pages to Ember 1.13.9, Ember-CLI 1.13.8, and upgrades the project and its dependencies according to the `ember init` command.

### CODE REVIEW

**Comment by elwayman02** [comment]
Nvm, running into new issues in testing:

```
Command failed: /bin/sh -c git checkout `git reflog HEAD | sed -n '/checkout/ !d; s/.* \(\S*\)$/\1/;p' | sed '2 !d'`
error: pathspec 'HEAD@{2}:' did not match any file(s) known to git.
error: pathspec 'checkout:' did not match any file(s) known to git.
error: pathspec 'moving' did not match any file(s) known to git.
error: pathspec 'from' did not match any file(s) known to git.
error: pathspec 'gh-pages' did not match any file(s) known to git.
error: pathspec 'to' did not match any file(s) known to git.
error: pathspec 'master' did not match any file(s) known to git.
```

Any idea what might have caused this?

**Comment by elwayman02** [comment]
This error does not seem to be preventing the package from working, however.  I was able to deploy my demo app by pushing the gh-pages branch as normal.  However, the fact that it's spewing this anyway is still concerning.

File: bower.json [hunk 1]
```diff
@@ -1,17 +1,16 @@
...
-    "ember-cli-shims": "stefanpenner/ember-cli-shims#0.0.3",
-    "ember-cli-test-loader": "rwjblue/ember-cli-test-loader#0.0.4",
-    "ember-load-initializers": "stefanpenner/ember-load-initializers#0.0.2",
-    "ember-qunit": "0.1.8",
-    "ember-qunit-notifications": "0.0.4",
-    "qunit": "~1.15.0"
+    "ember": "1.13.9",
+    "ember-cli-shims": "ember-cli/ember-cli-shims#0.0.3",
+    "ember-cli-test-loader": "ember-cli-test-loader#0.1.3",
+    "ember-data": "1.13.8",
```
Comment (mike-north) [inline bower.json]: no point in including ember-data

File: bower.json [hunk 2]
```diff
@@ -1,17 +1,16 @@
...
+    "ember-load-initializers": "ember-cli/ember-load-initializers#0.1.5",
+    "ember-qunit": "0.4.9",
+    "ember-qunit-notifications": "0.0.7",
+    "ember-resolver": "~0.1.18",
```
Comment (mike-north) [inline bower.json]: bump up to ~0.1.20 to get rid of a few ember 2.1.x deprecations

File: package.json [hunk 3]
```diff
@@ -18,27 +18,33 @@
...
-    "ember-cli-ic-ajax": "0.1.1",
-    "ember-cli-inject-live-reload": "^1.3.0",
-    "ember-cli-qunit": "0.1.2",
-    "ember-data": "1.0.0-beta.12",
-    "ember-export-application-global": "^1.0.0",
-    "express": "^4.8.5",
-    "glob": "^4.0.5"
+    "broccoli-asset-rev": "^2.1.2",
+    "ember-cli": "1.13.8",
+    "ember-cli-app-version": "0.5.0",
```
Comment (mike-north) [inline package.json]: bump to `^1.0.0`

File: package.json [hunk 4]
```diff
@@ -18,27 +18,33 @@
...
+    "ember-cli-content-security-policy": "0.4.0",
+    "ember-cli-dependency-checker": "^1.0.1",
+    "ember-cli-htmlbars": "0.7.9",
```
Comment (mike-north) [inline package.json]: `^1.0.0`

File: package.json [hunk 5]
```diff
@@ -18,27 +18,33 @@
...
+    "ember-cli-htmlbars-inline-precompile": "^0.2.0",
+    "ember-cli-ic-ajax": "0.2.1",
+    "ember-cli-inject-live-reload": "^1.3.1",
+    "ember-cli-qunit": "^1.0.0",
+    "ember-cli-release": "0.2.3",
+    "ember-cli-sri": "^1.0.3",
+    "ember-cli-uglify": "^1.2.0",
+    "ember-data": "1.13.8",
```
Comment (mike-north) [inline package.json]: remove ember-data

File: package.json [hunk 6]
```diff
@@ -18,27 +18,33 @@
...
+    "ember-disable-proxy-controllers": "^1.0.0",
+    "ember-export-application-global": "^1.0.3",
+    "ember-disable-prototype-extensions": "^1.0.0",
+    "ember-try": "0.0.6"
```
Comment (mike-north) [inline package.json]: `0.0.8`

**Comment by elwayman02** [comment]
@mike-north I have a follow-up PR for Ember 2.0 that bumps these package versions...I wanted to stick with the Ember-CLI defaults for now since this is already quite a version jump all around.

Also, here's a demo of an addon page deployed with this branch: http://elwayman02.github.io/ember-tumblr/

File: bower.json (hunk 1 above)
Comment (elwayman02) [inline bower.json]: :+1:

File: bower.json (hunk 2 above)
Comment (elwayman02) [inline bower.json]: Tilde versioning should take care of this

File: package.json (hunk 3 above)
Comment (elwayman02) [inline package.json]: Addressed in #31

File: package.json (hunk 4 above)
Comment (elwayman02) [inline package.json]: Addressed in #31

**Comment by knownasilya** [comment]
Is this finished?

**Comment by elwayman02** [comment]
This is incredibly outdated at this point, but yes it's been finished for a while.

**Pull request merged**